charts/kyverno/values.yaml

```diff
@@ -71,6 +71,14 @@ resources:
     cpu: 100m
     memory: 50Mi
 
+initResources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 10m
+    memory: 64Mi
```

Comment:
And we also need to add this to [definitions/manifest/deployment.yaml](https://github.com/kyverno/kyverno/blob/main/definitions/manifest/deployment.yaml).